LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file can be removed and it will be inherited from the shared repo at https://github.com/stylelint/.github/blob/main/LICENSE